resources/warnings.json

```diff
@@ -13035,7 +13035,7 @@
     "versions": [
       {
         "lastVersion": "2.1.10",
-        "pattern": ".*"
+        "pattern": "([01][.].*|2[.]0[.].*|2[.]1.[0-9]|2[.]1[.]10)"
```

Comment:
FTR our (sadly not public) script generates, based on currently published releases and the usual format that allows any suffixes,
```suggestion
        "pattern": "(1|2[.]0|2[.]1[.][0-9]|2[.]1[.]10)(|[.-].+)"
```

There's a bug in your expression with the non-escaped `.` in `2[.]1.[0-9]`. Version 2.100 would be considered vulnerable.

Reply:
:man_facepalming: Thanks!